lib/Feature/FeatureModel.cpp

```diff
@@ -109,7 +109,7 @@ void FeatureModelBuilder::detectXMLAlternatives() {
     std::vector<std::string> Frontier(Children[FeatureName].begin(),
                                       Children[FeatureName].end());
     while (!Frontier.empty()) {
-      const auto &FName = Frontier.back();
+      const auto &FName = std::string(Frontier.back());
```

Comment:
```suggestion
      const auto FName = std::string(Frontier.back());
```
We should use a value here, with a ref you get a reference to a temporary, which can be dangerous, for example, when someone removes the const because only const references extend the lifetime of a temporary.